packages/yourloops/components/patient-list/patient-list-header.tsx

```diff
@@ -147,6 +150,25 @@ export const PatientListHeader: FunctionComponent<PatientListHeaderProps> = (pro
                 {t('button-add-new-patient')}
               </Button>
             }
+            {
+              user.isUserHcp() && TeamUtils.isPrivate(selectedTeam) &&
+              <Tooltip
+                title={t('add-new-patient-disabled-info')}
+                placement="left"
+              >
+                <span data-testid="add-patient-button-disabled">
+                  <Button
+                    startIcon={<PersonAddIcon />}
+                    variant="contained"
+                    size="large"
+                    disableElevation
+                    disabled
+                  >
+                    {t('button-add-new-patient')}
+                  </Button>
+                </span>
+              </Tooltip>
+            }
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏I suggest another solution to avoid code duplication, the tooltip isn't displayed if the title is `undefined`. So there could be one button like this
```suggestion
{user.isUserHcp() &&
              <Tooltip
                title={!TeamUtils.isPrivate(selectedTeam) ? undefined : t('add-new-patient-disabled-info')}
                placement="left"
              >
                <span data-testid="add-patient-button-disabled">
                  <Button
                    startIcon={<PersonAddIcon />}
                    variant="contained"
                    size="large"
                    disableElevation
                    onClick={() => { setShowAddPatientDialog(true) }}
                    disabled={TeamUtils.isPrivate(selectedTeam)}
                  >
                    {t('button-add-new-patient')}
                  </Button>
                </span>
              </Tooltip>
            }
```